dace/codegen/targets/intel_fpga.py

```diff
@@ -987,6 +1010,32 @@ def generate_undefines(self, sdfg, dfg, node, callsite_stream):
                     callsite_stream.write("#undef {}".format(memlet_name),
                                           sdfg, sdfg.node_id(dfg), node)
 
+    def generate_converter(self, is_unpack, dtype, veclen, node, state_id,
+                           sdfg, function_stream):
+        if (is_unpack, dtype, veclen) in self.converters_generated:
+            return
+        if is_unpack:
+            function_stream.write(
+                """\
```

Comment:
```suggestion
                """
```

Reply:
Nah, the convention is that you clear space after yourself, not before yourself.